src/renderer/tvgShape.cpp

```diff
@@ -130,11 +130,11 @@ Result Shape::appendCircle(float cx, float cy, float rx, float ry) noexcept
     auto ryKappa = ry * PATH_KAPPA;
 
     pImpl->grow(6, 13);
-    pImpl->moveTo(cx, cy - ry);
-    pImpl->cubicTo(cx + rxKappa, cy - ry, cx + rx, cy - ryKappa, cx + rx, cy);
+    pImpl->moveTo(cx + rx, cy);
     pImpl->cubicTo(cx + rx, cy + ryKappa, cx + rxKappa, cy + ry, cx, cy + ry);
     pImpl->cubicTo(cx - rxKappa, cy + ry, cx - rx, cy + ryKappa, cx - rx, cy);
     pImpl->cubicTo(cx - rx, cy - ryKappa, cx - rxKappa, cy - ry, cx, cy - ry);
+    pImpl->cubicTo(cx + rxKappa, cy - ry, cx + rx, cy - ryKappa, cx + rx, cy);
```

Comment:
Please see line 218 as well; we can reuse this instead of creating an individual implementation.

Reply:
thx for tip with line 218! fixed.
I'm not sure what do you mean by 'reuse this'. here we have a circle. there is a rounded rectangle. I can add there an extra case for circles, but is it worth it?

Reply:
@hermet do you want me to make a special case for circles in appendRect and in the appendCircle call appendRect?

Reply:
@mgrudzinska Apologies for the confusion. I believe the appendRect() method should not be modified; the previous version seems correct. Please revert the changes made to appendRect(). Also, I had previously modified appendCircle() to start at 90 degrees, but I had to revert the change due to an issue caused by Lottie. I will double-check it, basically I agree on this change.

Reply:
@hermet I double checked in the svg standard. the rounded rect should start in a different point than it is in the current version of the code.  so both this changes are correct. 
but if they cause some other consequences lets investigate it and keep this pr open till then.

Reply:
@mgrudzinska Got it, Thanks. soon, I will double-check any breaks by this change.